src/test/kotlin/com/vanniktech/rxriddles/solutions/Riddle102Solution.kt

```diff
@@ -0,0 +1,14 @@
+package com.vanniktech.rxriddles.solutions
+
+import io.reactivex.Single
+import io.reactivex.functions.BiFunction
+import io.reactivex.schedulers.Schedulers
+
+object Riddle102Solution {
+  fun solve(first: Single<Int>, second: Single<Int>)
+      = Single.zip(
```

Comment:
I think formatting is different here from the other solutions. Can you double check?